deploy/files/config-development.env

```diff
@@ -6,8 +6,9 @@ PGUSER=atr
 PGPASSWORD=atr
 PGHOST=localhost
 PGPORT=5432
-ENVIRONMENT=dev
```

Comment:
@evmiguel I see `dev` being referenced in `embed.js` but I'm assuming `development` is being inferred throughout when this ENV var doesn't exist. Should those `embed.js` references to this var (or lack of) also be updated?

Reply:
@howard-e Good catch! Since the vagrant environment mirrors production, it's technically not using the `dev` configuration. I think we should name the variable something else with a different value.

Reply:
What would be the consequence of leaving that variable unchanged?

Reply:
@jugglinmike The app would look for the server side rendering files for the embed feature in a different directory.